### PR TITLE
Add some security pages

### PR DESCRIPTION
**Pull request by LukasReschke**
@jospoortvliet Please review.

### CODE REVIEW

**Comment by mar1u5** [comment]
:+1:

File: templates/footer.php [hunk 1]
```diff
@@ -28,14 +28,14 @@
...
-<!--             <li class="menu-user-manual"><a href="<?php echo $DOCUMENTATION_USER; ?>">User manual</a></li> -->
-<!--             <li class="menu-user-manual"><a href="<?php echo $DOCUMENTATION_DEVELOPER; ?>">Developer manual</a></li> -->
-<!--             <li class="menu-security"><a href="<?php get_bloginfo('url'); ?>/security/">Security</a></li> -->
+            <li class="menu-support"><a href="<?php get_bloginfo('url'); ?>/support/">Support</a></li>
+            <li class="menu-admin-manual"><a href="<?php echo $DOCUMENTATION_ADMIN; ?>">Admin manual</a></li>
+            <li class="menu-user-manual"><a href="<?php echo $DOCUMENTATION_USER; ?>">User manual</a></li>
+            <li class="menu-user-manual"><a href="<?php echo $DOCUMENTATION_DEVELOPER; ?>">Developer manual</a></li>
+            <li class="menu-security"><a href="<?php get_bloginfo('url'); ?>/security/">Security</a></li>
 <!--             <li class="menu-store"><a href="http://www.cafepress.com/nextcloudshop">Nextcloud Store</a></li> -->
 <!--          </ul>  -->
```
Comment (vincchan) [inline templates/footer.php]: @LukasReschke @jospoortvliet 
unordered list is not properly closed and indention is off

Comment (jospoortvliet) [inline templates/footer.php]: Fixed in https://github.com/nextcloud/nextcloud.com/commit/dd96fb56ae4692ec956157fa94afa1218b7ea2e4

Comment (vincchan) [inline templates/footer.php]: @jospoortvliet it's still not fixed :/

Comment (jospoortvliet) [inline templates/footer.php]: Refresh in 15 minutes, deploy takes some time, esp with the caching ;-)

Comment (vincchan) [inline templates/footer.php]: @jospoortvliet I wasn't talking about code in production :) 

anyways fixed in #32 :)

**Pull request merged**